gulpfile.js

```diff
@@ -7,8 +7,13 @@ var gulp = require('gulp'),
     outDir = tsconfig.compilerOptions.outDir || 'dist',
     testPattern = 'dist/**/spec/*.js';
 
-gulp.task('build', () => {
+gulp.task('copy:typings', () => {
```

Comment:
does it need to be copied? I thought whatever it needed would be exported during the typescript build. meh. i'm not too concerned about it.